icevision/models/torchvision/fastai_callbacks.py

```diff
@@ -2,7 +2,8 @@
 
 from icevision.imports import *
 from icevision.engines.fastai import *
-from icevision.models.torchvision import faster_rcnn
+
+# from icevision.models.torchvision import faster_rcnn
```

Comment:
Will we need this line at a later point or can we just delete it?

Reply:
Ahh good catch.. I will just remove it..